tests/spec/w3c/defaults-spec.js

```diff
@@ -74,4 +80,61 @@ describe("W3C — Defaults", () => {
     expect(rsConf.license).toBe("c0");
     expect(rsConf.specStatus).toBe("ED");
   });
+
+  it("doesn't show the W3C logo if no group or an invalid group is specified", async () => {
+    const ops = makeStandardOps({
+      specStatus: "WD",
+    });
```

Comment:
OCD: 
```suggestion
    const ops = makeStandardOps({ specStatus: "WD" });
```
(and ditto on rest)